src/lib/sort-rules.js

```diff
@@ -1,9 +1,29 @@
 'use strict'
 
 function getSortedRules(rules) {
-  return rules.sort(function sort(a, b) {
-    return a > b ? 1 : -1
-  })
+  return Array.isArray(rules) ? getSortedRulesArray(rules) : transformIntoSortedRulesArray(rules)
+}
+
+function getSortedRulesArray(rules) {
+  return rules.sort(sortAlphabetically)
+}
+
+function transformIntoSortedRulesArray(allRulesArray) {
+  var sortedRules = []
+
+  Object.keys(allRulesArray)
+    .sort(sortAlphabetically)
+    .forEach(function mapRules(ruleName) {
+      var singleRuleArray = {}
```

Comment:
And this is the kind of mistakes someone makes when getting distracted while coding :/